custom_typings/yeoman-environment.d.ts

```diff
@@ -4,7 +4,7 @@ declare module 'yeoman-environment' {
     registerStub(generator: any, namespace: string);
     run(target: string, options: any, done: Function);
     getGeneratorsMeta(): YeomanEnvironment.GeneratorMeta[];
-    lookup(callback: () => void);
+    lookup(callback: (error?: Error) => void);
```

Comment:
It's just that when JS code says "error", they often don't really mean `Error`. It's case-by-case. In this case they mean `null` :/